arch/arm64/boot/dts/qcom/msm8953.dtsi

```diff
@@ -2281,16 +2281,16 @@
 			status = "disabled";
 		};
 
-		i2c_6: i2c@7af6000 {
-			compatible = "qcom,i2c-qup-v2.2.1";
+		spi_6: spi@7af6000 {
+			compatible = "qcom,spi-qup-v2.2.1";
 			reg = <0x7af6000 0x600>;
```

Comment:
My downstream sources are sure that there's i2c_6 (BLSP2 QUP2) at this address.. Just for 8917 and 8937 it has an SPI at this address.

Can you give a reference to where the SPI definition is from?

Reply:
well, found it.. https://github.com/realme-kernel-opensource/realmeQ_realme5Pro-kernel-source/blob/master/arch/arm64/boot/dts/18097/msm8953.dtsi#L698

but still, don't replace the i2c definition but add a new spi one. Also at the same time feel free to add spi@78b7000 which I also see there

Reply:
I put all the details that I've foudn in the description of this PR